### ISSUE TITLE
how to use list-remote version filtering?

### ISSUE THREAD

**Issue by AnalogMemory** [Schniz/fnm#587]
Hello, thanks for building this excellent tool!

I was trying to see if there's a way to version filter when you run `fnm list-remote`

When i attempt `fnm list-remote 16` i get the following error

`error: Found argument '16' which wasn't expected, or isn't valid in this context`

It seems support for this was added in `v1.19.0` with #190 
But not sure what the correct options are to pass in

Happy to update the docs if they need it

**Comment by Schniz** [comment]
you are right, it is a regression with the migration from Reason to Rust. Need to rebuild it.
